packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -591,8 +624,13 @@ describe('CDK Include', () => {
 
     expect(cfnBucket.cfnOptions.updatePolicy).toBeDefined();
 
-    expect(stack).toMatchTemplate(
-      loadTestFileToJsObject('resource-attribute-update-policy.json'),
+    // convert string values to boolean equivalents to appease test framework
+    const template = loadTestFileToJsObject('resource-attribute-update-policy.json');
+    template.Resources.Bucket.UpdatePolicy.AutoScalingReplacingUpdate.WillReplace = false;
+    template.Resources.Bucket.UpdatePolicy.EnableVersionUpgrade = true;
+
+    Template.fromStack(stack).templateMatches(
+      template,
```

Comment:
Can we instead change to an assertion on the UpdatePolicy attribute, like the test name says?